prompt2model/dataset_generator/__init__.py

```diff
@@ -1,10 +1,10 @@
 """Import DatasetGenerator classes."""
 from prompt2model.dataset_generator.base import DatasetGenerator, DatasetSplit
 from prompt2model.dataset_generator.mock import MockDatasetGenerator
-from prompt2model.dataset_generator.openai_gpt import OpenAIDatasetGenerator
+from prompt2model.dataset_generator.prompt_based import PromptBasedDatasetGenerator
```

Comment:
APIBasedDatasetGenerator?

Reply:
The dataset generator could probably also use an open-source model not through an online API, so maybe PromptBasedDatasetGenerator is OK?